src/pages/Login.js

```diff
@@ -0,0 +1,77 @@
+import React, {useRef, useEffect, useState} from 'react';
+import MainLayout from '../components/MainLayout';
+import { useForm } from 'react-hook-form';
+import './User.scss';
+
+const Login = () => {
+  const nameRef = useRef();
+  const { register, handleSubmit } = useForm()
+  const [err, setErr] = useState(' ')
+
+  useEffect(() => {
+    document.title = 'Login';
+	  nameRef.current.focus();
+  }, []);
+
+  const handleFormSubmit = (data) => {
+	  setErr('')
+	  fetch(process.env.REACT_APP_API_BASE + 'signin', {
+      method: 'POST',
+	    body: JSON.stringify(data),
+	    headers: {
+		  'Content-Type': 'application/json'
+	    }
+	  })
+	  .then(res => res.json())
+	  .then(data => {
+	    if (data.status === 404) {
+	      //invalid login credentials
+        setErr('Username or password incorrect')
+	    } else if (data.status === 400) {
+		    //invalid request
+		    setErr('Invalid request')
+	    } else if(data.status === 200) {

```

Comment:
Just a reminder: Depending on the role of the user, they will be able to observe or perform some functions. (editors and administrators).